### ISSUE TITLE
Actions before load can "stick" to `onPlay` and cause uninteded behavior

### ISSUE THREAD

**Issue by terotil** [goldfire/howler.js#76]
When sound is not loaded actions are postponed by adding callbacks. All of them are clearly ment to be executed only once (e.g. volume adjustment before sound is loaded). However, when hooked to event that is possibly received more than once (specifically `onPlay`), they can get executed several times, and lead to confusing behavior.  e.g. sound created this way `new Howl(...).pause()` won't play at all, because `pause()` will stick permanently to `onPlay` (see http://jsfiddle.net/VGV85/1/).

It appears that all "postponed" actions are inside `if (!self._loaded) { ... }`, but some of them are still hooked to `onPlay` instead of `onLoad`.
